runtime/parser.c

```diff
@@ -498,9 +498,11 @@ SanitizeMsg(smsg_t *pMsg)
 					case '\b':
 						pDst[iDst++] = 'b';
 						break;
+#ifndef __xlc__
```

Comment:
doesn't that break configs between AIX and non-aix?

Reply:
The espaced char '\e' is not understood by xlc.

Reply:
ah, ok - maybe we need to do work-around - how about '\x1b`?